apix_documentation_swagger_v3.yaml

```diff
@@ -351,6 +326,130 @@ info:
+    familiarized with their responses. This will give you an idea of how to
+    properly construct requests in your application (and save you coding time).
+
+    You can do so by using the following dynamic sections of this documentation.
+    In order for the endpoints to work properly, first you have to click on the
+    `Authorize` button (located before the endpoints, to the right) and enter
+    your api-key and secret-key corresponding to the environment you wish to use
+    (make sure that the correct environment is selected on the `Server` combo
+    box located below and to the left of the `Authorize` button). Alternatively,
+    you can click on the 'lock' icon located on each endpoint to enter your
+    keys.
+
+    Once you have authenticated, you can now click on each endpoint's
+    `Try it out` button, which will let you enter the parameters' values (if
+    any) and display a big blue `Execute` button.
+
+    Additionally, you can use any of the following API-testing tools:
+
+      * [Paw](https://paw.cloud/)
+      * [Postman](https://www.getpostman.com/)
+      * [Insomnia](https://insomnia.rest/)
+
+    ## Handling Multi-factor Authentication (MFA)
+
+    Some billers require more than just a username/password to log in. In
+    those situations, arcus will associate one or more MFA challenges with a
+    bill. An MFA challenge can be a verification code sent to a phone, a
+    secret question, an image, a list of images to choose from, or different
+    choices which only the end user would know the answer to. When arcus
+    refreshes a bill, it will be set into a status of waiting, which means
+    that the end user must answer a challenge.
+
+    ## Forcing errors and MFAs
+
+    Within the Sandbox, we have created a merchant that will allow you to force
+    a failed response as well as test the usage of MFA Challenges:
+      - name: Arcus Faker
+      - ID: 451cc3f4-68c4-4280-9d8a-164891bbd850
+
+    This can be done by using a specific `login` value in those endpoints that
+    create a resource for xData or xChange. If you're testing MFA challenge
+    behavior, depending on which product you're using you will have to provide
+    the fixed answer (see table below) on the corresponding `PATCH` endpoint:
+    <b>PATCH/bills/{id}</b> for xData or <b>PATCH/migrations/{id}</b> for
+    xChange.
+
+    Following is the list of the existing `login` values for testing purposes:
+
+    <table>
+      <tr>
+        <th>Login value</th>
+        <th>Explanation</th>
+      </tr>
+      <tr>
+        <td>question.the_answer</td>
+        <td>
+          This will associate a new MFA Challenge of type 'question' with the
+          bill. Challenge answer: the_answer
+          Take into account that `the_answer` can actually be any string you
+          want (e.g.: `llama`, `mr alpaca`, `Arcus Fi`, etcetera), just make
+          sure that your answer does not include dots.
```

Comment:
Now dots are supported for the answers: https://github.com/regalii/plunderer/pull/162

Reply:
Done ;)